File: C/section07/S07E01.c
#include <stdio.h>
#include <stdbool.h>

int main() {
    int number;
    int biggestNumber = 0;

    while(number != 0) {
    printf("Please enter an integer number: ");
    scanf("%d", &number);
    if (number > biggestNumber) {
        biggestNumber = number;
    }
}   
    printf("The biggest number was %d.", biggestNumber);

    return 0;
}

File: Python/section07/S07e01.py
number = -1
biggestNumber = 0

number = int(input("Please enter a number, enter 0 to stop: "))
while number != 0:
    
    if number > biggestNumber:
        biggestNumber = number
    number = int(input("Please enter a number, enter 0 to stop: "))
    
print("\nThe biggest entered number was: {0}".format(biggestNumber))